pandas/core/sparse/frame.py

```diff
@@ -95,6 +95,20 @@ def __init__(self, data=None, index=None, columns=None, default_kind=None,
                                  dtype=dtype, copy=copy)
         elif isinstance(data, DataFrame):
             mgr = self._init_dict(data, data.index, data.columns, dtype=dtype)
+        elif isinstance(data, Series):
+            if columns is None:
+                if data.name is None:
+                    raise ValueError('cannot pass a series '
+                                     'w/o a name or columns')
```

Comment:
I don't think you need any of this, you can simply all ``to_manager`` with the column (in a list)